odh/overlays/moc/monitoring/dashboards/operate-first/argocd.yaml

```diff
@@ -3852,4 +3852,4 @@ spec:
       "uid": "97AYZMTGk",
       "version": 1
     }
-  name: argocd.json
\ No newline at end of file
+  name: argocd.json
```

Comment:
Please undo this change.

Reply:
Oh, I see.. this one is breaking the pre-commit check. Don't mind my comment. It seems multiple people are running into this...

Reply:
I removed this from this #PR ... as you requested ... you can create another small PR to fix only this ... this is not related to this PR so make sense not to include ... :-)